tests/headers/derive-debug-opaque-template-instantiation.hpp

```diff
@@ -0,0 +1,10 @@
+// bindgen-flags: --force-derive-debug
+
+template<typename T, int N>
+class Opaque {
```

Comment:
Ahh nice to know, I overseved the behavior with all the other tests while developing. But I could not find the right position in the code where this case is handled.